spec/models/manageiq/providers/embedded_terraform/automation_manager/stack_spec.rb

```diff
@@ -30,4 +30,161 @@
       end
     end
   end
+
+  describe "#raw_stdout" do
+    let(:stack) { FactoryBot.create(:terraform_stack, :miq_task => miq_task) }
+    let(:template) { FactoryBot.create(:terraform_template) }
+
+    context "when miq_task.job present" do
+      let(:terraform_runner_url) { "https://1.2.3.4:7000" }
+      let(:hello_world_retrieve_response) do
+        require 'json'
+        JSON.parse(File.read(File.join(__dir__, "../../../../../lib/terraform/runner/data/responses/hello-world-retrieve-success.json")))
```

Comment:
We might want to consider moving this up to `spec/data` or something if it is going to be used by more than just the `Terraform::Runner` specs since this `../../../etc...` makes it hard to read where this file lives

Reply:
Yes agree makes it harder to read,  
could make it like 
"../../../../../../spec/lib/terraform/runner/data/responses/hello-world-retrieve-success.json"

also maybe should move this common test data,  to somewhere like `<base-dir>/spec/data`.  then still look like
"../../../../../../spec/data/responses/hello-world-retrieve-success.json"

Reply:
don't know if there more easier way to refer to test data resources without using "../../../../../../"

Reply:
We can use the `ManageIQ::Providers::EmbeddedTerraform::Engine.root` Pathname to build a path from the root of the plugin

Reply:
ok